moncli/entities/column_value.py

```diff
@@ -262,6 +263,68 @@ def format(self):
             return result
         return self.null_value
         
+class DependencyValue(ColumnValue):
+    """An Dependency column value.
+    
+    Properties
+
+        item_ids : `list[str]`
+            The list of linked items unique identifiers.
+
+    Methods
+
+        add_item : `void`
+            Add item to link list.
+        remove_item : `void`
+            remove item from remove.
+
+    """
+    def __init__(self, **kwargs):
+        super(DependencyValue, self).__init__(**kwargs)
+    
+    @property
+    def item_ids(self):
+        """List of linked items unique identifiers."""
+        try:
+            return [str(id['linkedPulseId']) for id in loads(self.value)['linkedPulseIds']]
+        except:
+            return []
+
+    @property
+    def settings(self):
+        return loads(self.__settings)
+
+    def add_item(self, item_id: str):
+        """Add item to link list.
+
+        Parameters
+
+            item_id : `str`
+                Item unique identifier to add.
+        """
+        ids = self.item_ids
+        ids.append(str(item_id))
+        self.value = dumps({'linkedPulseIds': [{'linkedPulseId': str(id)} for id in ids]})
+
+    def remove_item(self, item_id: str):
+        """Remove item from link list.
+
+        Parameters
+
+            item_id : `str`
+                Item unique identifier to remove.
+        """
+
+        if item_id not in self.item_ids:
+            raise ItemIdNotFound(item_id)
+
+        ids = [id for id in self.item_ids if id != item_id]
+        self.value = dumps({'linkedPulseIds': [{'linkedPulseId': int(id)} for id in ids]})
+
+    def format(self):
```

Comment:
This formatting will not work.  Please refer to how the ItemLinkValue handles the format() method with the code below:

`return {'item_ids': [int(id) for id in self.item_ids]}`